packages/react/src/hooks/useInputState/useInputState.spec.tsx

```diff
@@ -0,0 +1,157 @@
+import { describe } from 'node:test';
+import { useInputState } from '.';
+import { expect, it } from 'vitest';
+import { renderHook, screen } from '@testing-library/react';
+import { renderSetup } from '../../_internal/test/renderSetup';
+import { userEvent } from '@testing-library/user-event';
+
+const TestComponent = ({
+  initial,
+  isObject,
+  isName,
+}: {
+  initial: any;
+  isObject: boolean;
+  isName: boolean;
+}) => {
+  const { value, onChange, reset } = useInputState(initial);
+
+  return (
+    <>
+      <h1 role="test-value">{isObject ? value.test : value}</h1>
+      {isName ? (
+        <input
+          data-testid="test-input"
+          name="test"
+          onChange={onChange}
+          value={isObject ? value.test : value}
+        />
+      ) : (
+        <input
+          data-testid="test-input"
+          onChange={onChange}
+          value={isObject ? value.test : value}
+        />
+      )}
+      <button onClick={reset}>reset</button>
+    </>
+  );
+};
+
+describe('useInputState', () => {
+  it('value값이 단일 값일 때, name이 없어도 값이 변경됩니다.', async () => {
+    renderSetup(
+      <TestComponent
+        initial={'initial value'}
+        isObject={false}
+        isName={false}
+      />
+    );
+
+    const value = screen.getByRole('test-value');
+    const input = screen.getByTestId('test-input');
```

Comment:
```suggestion
    const input = screen.getByRole('textbox');
```
input태그의 role은 textbox로 변경해주세요
쿼리 사용 시 role이 가장 우선순위가 높습니다!

https://testing-library.com/docs/queries/about/#priority